docs/containers/quickstart-python.md

```diff
@@ -0,0 +1,142 @@
+---
+Order: 4
+Area: containers
+TOCTitle: Python
+ContentId: 3a9bc520-95e2-416e-a0ac-5be02a38c4c3
+PageTitle: Build and run a Python app in a container
+DateApproved: 2/18/2020
+MetaDescription: Develop, build, and debug a Python app in a Docker container, using Visual Studio Code.
+---
+# Python in a container
+
+In this guide you will learn how to:
+
+- Create a `Dockerfile` file describing a simple Python container.
+- Build, run, and verify the functionality of a [Django](https://www.djangoproject.com/), [Flask](http://flask.pocoo.org/), or General Python app.
+- Debug the app running in a container.
+
+## Prerequisites
+
+- Docker Desktop and the VS Code Docker extension must be installed as described in the [overview](overview.md#installation).
+- For Python development, complete all [Getting started with Python](/docs/python/python-tutorial.md) steps
+- A runnable Python application
+
+### Create a Python project
+
+If you don't have a Python project already, follow these commands sequentially from the terminal:
+
+```bash
+pip install django
+django-admin startproject helloworld
+cd helloworld
+code .
+```
+
+If you want to containerize a complete Django or Flask web app, you can use one of the following samples:
+
+- [python-sample-vscode-django-tutorial](https://github.com/microsoft/python-sample-vscode-django-tutorial/tree/tutorial), which is the result of following the [Django Tutorial](/docs/python/tutorial-django.md)
+
+- [python-sample-vscode-flask-tutorial](https://github.com/microsoft/python-sample-vscode-flask-tutorial/tree/tutorial), which is the result of following the [Flask Tutorial](/docs/python/tutorial-flask.md)
+
+After verifying your app runs properly, you can now Dockerize your application.
+
+## Add Docker files to the project
+
+1. Open the project folder in VS Code.
+2. Open the Command Palette (`kb(workbench.action.showCommands)`) and use **Docker: Add Docker Files to Workspace...** command:
```

Comment:
We usually use the numeral 1 for all list items - that way you can add/substract items over time and the published list remains ordered without any manual work.

Reply:
Yes, this is something I wanted to do, however for some reason with putting the **Tip**: in between 5 and 6, it breaks the numbering. I know this is not ideal, but I do not know a valid alternative

Reply:
But I did go ahead and update any other list where possible to be all 1s. Thanks for that suggestion

Reply:
If you indent the Tip, so that the beginning of that block aligns with the first text in the previous number, it will be a child of the item and won't break the list.